src/MongoDB.Driver/IAggregateFluent.cs

```diff
@@ -206,6 +206,27 @@ public interface IAggregateFluent<TResult> : IAsyncCursorSource<TResult>
         /// <returns>The fluent aggregate interface.</returns>
         IAggregateFluent<TNewResult> Lookup<TForeignDocument, TNewResult>(string foreignCollectionName, FieldDefinition<TResult> localField, FieldDefinition<TForeignDocument> foreignField, FieldDefinition<TNewResult> @as, AggregateLookupOptions<TForeignDocument, TNewResult> options = null);
 
+        /// <summary>
+        /// Appends a lookup stage to the pipeline.
+        /// </summary>
+        /// <typeparam name="TForeignDocument">The type of the foreign document.</typeparam>
+        /// <typeparam name="TAsElement">The inner type of <typeparamref name="TAs" /> collection.</typeparam>
+        /// <typeparam name="TAs">The type of <typeparamref name="TAs" /> collection.</typeparam>
+        /// <typeparam name="TNewResult">The type of the new result.</typeparam>
+        /// <param name="foreignCollection">The other collection.</param>
```

Comment:
Let's use "foreign" instead of "other":

    /// <param name="foreignCollection">The foreign collection.</param>